esmvaltool/recipes/ref/recipe_seaborn_trend_regions.yml

```diff
@@ -0,0 +1,309 @@
+      decomposed: true
+      ids:
+        Acronym:
+          - 'NWN'
+          - 'NEN'
+          - 'GIC'
+          - 'WNA'
+          - 'CNA'
+          - 'ENA'
+          - 'NCA'
+          - 'SCA'
+          - 'CAR'
+          - 'NWS'
+          - 'NSA'
+          - 'NES'
+          - 'SAM'
+          - 'SWS'
+          - 'SES'
+          - 'SSA'
+          - 'NEU'
+          - 'WCE'
+          - 'EEU'
+          - 'MED'
+          - 'SAH'
+          - 'WAF'
+          - 'CAF'
+          - 'NEAF'
+          - 'SEAF'
+          - 'WSAF'
+          - 'ESAF'
+          - 'MDG'
+          - 'RAR'
+          - 'WSB'
+          - 'ESB'
+          - 'RFE'
+          - 'WCA'
+          - 'ECA'
+          - 'TIB'
+          - 'EAS'
+          - 'ARP'
+          - 'SAS'
+          - 'SEA'
+          - 'NAU'
+          - 'CAU'
+          - 'EAU'
+          - 'SAU'
+          - 'NZ'
+          - 'WAN'
+          - 'EAN'
+    area_statistics:
+      operator: mean
+    linear_trend:
+    convert_units:
+      units: m s-1 yr-1
+
+  trend_hus200:
+    <<: *trend200
+    convert_units:
+      units: year-1
+
+
+diagnostics:
+  tas_trends:
+    variables:
+      tas:
+        preprocessor: trend_tas
+        mip: Amon
+        timerange: 1980/2009
+        additional_datasets:
+          - {dataset: HadCRUT5, project: OBS, type: ground,
+             version: 5.0.1.0-analysis, tier: 2}
+          - {dataset: ERA5, project: native6, type: reanaly, version: v1,
+             tier: 3}
+    scripts:
+      plot:
+        script: seaborn_diag.py
+        seaborn_func: barplot
+        seaborn_kwargs:
+          x: shape_id
+          y: tas
+          hue: dataset
+        add_aux_coords: true
+        facets_as_columns: [dataset]
+        legend_title: Data
+        plot_object_methods:
+          set_xlabel: region
+          set_ylabel: tas trend [K/yr]
+        seaborn_settings:
+          style: ticks
+          rc:
+            figure.figsize: [28, 6]
+        suptitle: Trend in Near-Surface Air Temperature
+
+  pr_trends:
+    variables:
+      pr:
+        preprocessor: trend_pr
+        mip: Amon
+        timerange: 1980/2009
+        additional_datasets:
+          - {dataset: GPCP-SG, project: OBS, type: atmos, version: 2.3, tier: 2,
+             reference_for_metric: true}
+          - {dataset: ERA5, project: native6, type: reanaly, version: v1,
+             tier: 3, reference_for_metric: true}
+    scripts:
+      plot:
+        script: seaborn_diag.py
+        seaborn_func: barplot
+        seaborn_kwargs:
+          x: shape_id
+          y: pr
+          hue: dataset
+        add_aux_coords: true
+        facets_as_columns: [dataset]
+        legend_title: Data
+        plot_object_methods:
+          set_xlabel: region
+          set_ylabel: pr trend [mm kg m-3 day-1 yr-1]
+        seaborn_settings:
+          style: ticks
+          rc:
+            figure.figsize: [28, 6]
+        suptitle: Trend in Precipitation
+
+
+  psl_trends:

```

Comment:
Does it make sense to show the trends for psl? Might be difficult to interpret?

Reply:
I put the trend for the same variables as the time series and annual cycles. I think for regional analysis it is ok, to look at psl? But I could also skip it/use other variables if you prefere?

Reply:
@axel-lauer and I had a discussion about the plot, and it seemed a little hard to interpret for the different regions. But for consistency with the other recipes, I would say we leave it it for now, and then see what kind of feedback we get for the REF diagnostics.